src/Launcher.js

```diff
@@ -29,7 +29,7 @@ const {helper, assert, debugError} = require('./helper');
 const debugLauncher = require('debug')(`puppeteer:launcher`);
 const {TimeoutError} = require('./Errors');
 const WebSocketTransport = require('./WebSocketTransport');
-const PipeTransport = require('./PipeTransport');
+const {PipeTransport} = require('./PipeTransport');
```

Comment:
did this change because the majority of files do this and I think named imports in ESM are nicer than defaults so this aligns more closely with where I see the codebase going.

Reply:
+1 named exports 4 lyfe